<launch>

    <!-- Initialise the HSV filter -->
    <node name="HSVFilter_green_light" pkg="sb_vision" type="hsv_filter" output="screen">
        <!-- Input -->

        <!-- Output -->

        <!-- Node parameters -->
        <rosparam param="config_file" subst_value="true"> $(find sb_vision)/launch/green_light_filter.conf </rosparam>
        <rosparam param="update_frequency"> 5 </rosparam>
        <rosparam param="show_image_window"> true </rosparam>
        <rosparam param="show_calibration_window"> true </rosparam>

        <!-- Input image parameters if set manually -->
        <!--
        <rosparam param="image_width"></rosparam>
        <rosparam param="image_height"></rosparam>
        --> 
    </node>

    <!-- Initialise stoplight detection node  -->
    <node name="stoplight_detection" pkg="sb_vision" type="circle_detection" output="screen">
        <!-- Input -->

        <!-- Output -->

        <!-- Configuration parameters -->
        <rosparam param="minimum_target_radius">50</rosparam>
        <rosparam param="show_image_window">false</rosparam>
    </node>

</launch>
